uptane/common.py

```diff
@@ -117,6 +131,7 @@ def sign_signable(
     signable['signatures'].append(sign_over_metadata(
         signing_key, signable['signed'], datatype=datatype,
         metadata_format=metadata_format))
+    keyids_that_already_signed.append(signing_key['keyid'])
```

Comment:
In the comment block that starts on line 129, I would consider stating that the the next signing key in `keys_to_sign_with` is processed, if needed.  The current signing key is added to `keyid_that_already_signed` to prevent duplicates.

Reply:
Done here: 71704e8. Thanks.